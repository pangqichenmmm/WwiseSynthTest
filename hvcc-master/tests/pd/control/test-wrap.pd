#N canvas 632 324 310 286 10;
#X obj 36 49 loadbang;
#X obj 36 189 wrap;
#X msg 182 129 -0.5;
#X msg 149 129 0;
#X msg 107 129 0.75;
#X msg 220 129 -10.234;
#X obj 36 79 t b b b b b;
#X obj 36 219 print;
#X msg 36 129 1000.92;
#X obj 169 213 dac~;
#X connect 0 0 6 0;
#X connect 1 0 7 0;
#X connect 2 0 1 0;
#X connect 3 0 1 0;
#X connect 4 0 1 0;
#X connect 5 0 1 0;
#X connect 6 0 8 0;
#X connect 6 1 4 0;
#X connect 6 2 3 0;
#X connect 6 3 2 0;
#X connect 6 4 5 0;
#X connect 8 0 1 0;
