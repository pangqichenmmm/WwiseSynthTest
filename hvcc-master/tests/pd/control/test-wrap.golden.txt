[@ 0.000] print: 0.766
[@ 0.000] print: 0.5
[@ 0.000] print: 0
[@ 0.000] print: 0.75
[@ 0.000] print: 0.919983
